make/Makefile.operator.mk

```diff
@@ -131,7 +131,7 @@ purge-all-crs: .ensure-oc-login
 	  done
 
 ## run-operator: Runs the OSSM Console Operator via the ansible-operator.
-run-operator: install-crd install-cr get-ansible-operator
```

Comment:
See, here is where you need to force the CRD because this task DOES install/run the operator so the CRD is needed. That's why this is here.